src/utils/random-utils.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Get an array of numbers
+ * @param {number} n Length of the array
+ */
+export const getNumberArray = n => Array.from(Array(n).keys());
+
+/**
+ * Get a random number between 0 to n-1, inclusive
+ * @param {number} n Max number
+ */
+export const randomIndex = n => Math.floor(Math.random() * n);
```

Comment:
Again I think it's a good idea to swap out the `Math.random` calls here to use the seeded random function directly instead.

Reply:
@bru5 Cheers, I've updated this, let me know what you think